lotion/lib/tendermint.js

```diff
@@ -3,6 +3,20 @@ const { stringify } = require('deterministic-json');
 const { join } = require('path');
 const tendermint = require('tendermint-node');
 
+const CONSENSUS_PARAMS = {
```

Comment:
why do we need these? Aren't they autogenerated along with new genesis?

Reply:
I see... ` time_iota_ms` it is

Reply:
maybe add comment here to outline that we these are defaults except `time_iota_ms` ?

Reply:
`time_iota_ms` of `1000` is the default. But it seems that it only makes a difference if we have more than one validator proposing.

Reply:
I just added those for convenience in the future 😄